qubes-rpc/qvm-template-repo-query

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
+
+OPTS=()
+SPEC=
+while IFS= read -r line; do
+    if [ "$line" = "---" ]; then
+        break
+    fi
+    case "$line" in
+        --enablerepo=*|\
+        --disablerepo=*|\
+        --repoid=*|\
+        --repofrompath=*|\
```

Comment:
This one is not mentioned in the design doc, and also I don't think it's necessary.

Reply:
I think I was considering whether to add it, also decided it wasn't necessary, but forgot to remove the line. Done.